targets/unity-v2/templates/InstanceAPI.cs.ejs

```diff
@@ -58,7 +58,7 @@ namespace PlayFab
         /// </summary>
         public bool IsClientLoggedIn()
         {
-            return PlayFabHttp.IsClientLoggedIn();
+            return authenticationContext == null ? false : authenticationContext.IsClientLoggedIn();
```

Comment:
What does an instance without an authenticationContext even mean?

Reply:
I think we talked already about the same thing in C#. One of the goals of the initial design was to provide a smooth upgrade path for existing customers from static APIs to instance APIs. Their code is not instrumented to use authentication contexts, but they can easily create instances of APIs using a default constructor. In that case authentication context is set to null (not instantiated by default!). It allows us to fallback to PlayFabSettings values when calling APIs, and everything just works as it was before with static APIs. BUT, if a customer decides to use authentication context explicitly, they can set it, and then it will be used for authentication.

---
In reply to: [262255530](https://github.com/PlayFab/SDKGenerator/pull/420#discussion_r262255530) [](ancestors = 262255530)